src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -992,58 +1009,160 @@ public static void onWorldLoad(@NotNull final World world)
         }
     }
 
-    public static boolean backupColonyData()
+    /**
+     * Initiates a timer to schedule the colony backup task
+     * backupDelay of 0 disables automatic backup
+     */
+    private static void scheduleColonyBackup()
     {
-        if (numWorldsLoaded > 0 && saveNeeded)
+        final int backupDelay = Configurations.administration.colonyBackupTimeMins;
+        if (backupDelay > 0)
         {
-            saveColonies(false);
+            backupTimerTask = new BackupColonyTimerTask();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'backupTimerTask' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)